### ISSUE TITLE
NPM version 2.0.0 not working with nocha

### ISSUE THREAD

**Issue by colltoaction** [tj/should.js#138]
Hi, I just had a problem with your npm package, which seems to be recently updated.

I installed mocha and should, and it says

```
module.js:340
    throw err;
          ^
Error: Cannot find module './lib/should'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
```

If I install version 1.3.0 it goes well.

**Comment by diaswrd** [comment]
Same problem here, after update my should npm install..

```
module.js:340
    throw err;
          ^
Error: Cannot find module './lib/should'
  at Function.Module._resolveFilename (module.js:338:15)
  at Function.Module._load (module.js:280:25)
  at Module.require (module.js:362:17)
  at require (module.js:378:17)
  at Object.<anonymous> (/Users/wdias/dev/web/node_modules/should/index.js:2:18)
  at Module._compile (module.js:449:26)
  at Object.Module._extensions..js (module.js:467:10)
  at Module.load (/usr/local/lib/node_modules/coffee-script/lib/coffee-script/coffee-script.js:211:36)
  at Function.Module._load (module.js:312:12)
  at Module.require (module.js:362:17)
  at require (module.js:378:17)
```

**Comment by btd** [comment]
Yes, it seems problem in .npmignore. I will fix it soon.

**Comment by btd** [comment]
I published 2.0.1. It should be fixed now.
